docs/blocks/custom-fluids.md

```diff
@@ -0,0 +1,851 @@
+        "burn_odds": 0,
+        "flame_odds": 0
+      }
+    },
+    "events": {
+      "fluids:flow": {
+        "sequence": [
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, 1, 'template2') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, -1, 'template2')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~-1 ~ air run setblock ~ ~-1 ~ fluids:fluid_template_down" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, 1, 'template2') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, -1, 'template2')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~-1 ~ fluids:fluid_template1 run setblock ~ ~-1 ~ fluids:fluid_template_down" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, 1, 'template2') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, -1, 'template2')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~-1 ~ fluids:fluid_template2 run setblock ~ ~-1 ~ fluids:fluid_template_down" ]
+            }
+          },
+          {
+            "condition": "query.block_neighbor_has_any_tag(1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, 1, 'template2') || query.block_neighbor_has_any_tag(-1, 0, 0, 'template2') || query.block_neighbor_has_any_tag(0, 0, -1, 'template2')",
+            "run_command": {
+              "target": "self",
+              "command": [ "execute if block ~ ~-1 ~ fluids:fluid_template3 run setblock ~ ~-1 ~ fluids:fluid_template_down" ]
+            }
+          },
+
+          {
+            "condition": "!query.block_neighbor_has_any_tag(1, 0, 0, 'template2') && !query.block_neighbor_has_any_tag(0, 0, 1, 'template2') && !query.block_neighbor_has_any_tag(-1, 0, 0, 'template2') && !query.block_neighbor_has_any_tag(0, 0, -1, 'template2')",
+            "run_command": {
+              "target": "self",
+              "command": [ "setblock ~ ~ ~ air" ]
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+</Spoiler>
+
+## Falling Fluid Block
+
+Below is the JSON for the `falling fluid block`. To replicate the json, copy and quick replace `template` with your fluid's name. If this block detects air below it, it will place another `falling fluid block`. However, if it detects another block beneath it, it will behave like the `source fluid block`.
+
+<CodeHeader>BH\blocks\fluid_template\fluid_template_down.json</CodeHeader>
```

Comment:
Please make the codeheader follow the same style guide, `BP/blocks/filename.json`